src/Components/PeoplaPage/PeoplePage.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../People';
+import PeopleTable from '../PeopleTable/PeopleTable';
+
+const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    const fetchPeoples = async () => {
+      const response = await getPeople();
+
+      setPeople(response);
+    };
+
+    try {
+      fetchPeoples();
+    } catch (error) {
+      throw new Error(`${error}`);
```

Comment:
And you don't catch this `error`, create some state value like `requestError` and use conditional rendering to show it